protocol/test/presence.ex

```diff
@@ -1,6 +1,6 @@
-struct TestSchema.NotPresent
+struct TestSchema.Presence
 
-name "not_present"
+name "presence"
```

Comment:
This schema's name was important as `not_present` because that's the error message it was designed for. "Presence" isn't a Speck concept. It's ok for the test(s) around this PR to specifically target the bug case. It could be something like "boolean false is a valid value", or "falsy values coerce successfully". In the case of the latter you could do things like `0`, `""`, etc.

If you're describing what's in the schema, something like `AllTypesRequired` is pretty descriptive. I'm fine with catering a schema file to a narrow group of tests though, so it's ok to have schemas with different names but the same attributes. That makes it easier to change one schema without affecting other tests.

Reply:
Fixed in bacaf6c31df80a2f227c821f6aa63b97869780b7

I also left in the test for `TestSchema.NotPresent`. The schema existed, but there wasn't a test for it which is why my original intent was to use `Presence` for both `not_present` and `all_present` scenarios but I see how that can be confusing.

Reply:
It was probably my mistake forgetting to put those tests in, so thanks for fixing that!